client/components/popover/index.jsx

```diff
@@ -260,6 +274,27 @@ class Popover extends Component {
 	}
 
 	/**
+	 * Adjusts positition swapping left and right values
+	 * when right-to-left directionality is found.
+	 *
+	 * @param  {String} position Original position
+	 * @return {String}          Adjusted position
+	 */
+	adjustRtlPosition( position ) {
+		if ( this.props.isRtl ) {
+			switch ( position ) {
+				case 'top right': return 'top left';
```

Comment:
Added some _extra_ spacing _and_ decided to be nice and add the non-canonical `left`/`right`-first positions. Thanks!